scripts/tokenomics-emulation/config.json

```diff
@@ -113,7 +113,8 @@
       {
         "name": "send-ddc-transaction",
         "amount": 1,
-        "sendData": "This is test data"
+        "string_lenght": 10,
```

Comment:
We need bytes. It should be 128 by default.

Reply:
Updated